lesson3/news/index.js

```diff
@@ -0,0 +1,59 @@
+const http = require('http')
+const rp = require('request-promise')
+const cheerio = require('cheerio')
+
+const newsServiceUrl = 'http://www.newsru.com'
+
+const populateServerRespondNews = res => {
+  const news = []
+  const promises = []
+
+  rp(`${newsServiceUrl}/allnews`)
```

Comment:
I'd recommend using another one promise lib instead of native. Please take a look at [this](http://bluebirdjs.com/docs/getting-started.html).

Reply:
It will make your code more clear and understandable

Reply:
Yes, I agree. I saw this library and found it very interesting, just didn't have enough time to discover it closer. Will do it necessary soon. Thank you.